pact/coin-contract/v2/coin.repl

```diff
@@ -556,7 +559,20 @@
     (format "Account: {} Guard: {}" [(at 'balance i) (at 'guard i)])))
 
 ; account details will now feature rotated guard
-(rotate 'emily (read-keyset 'doug))
+(env-keys ["keys1", "keys2"])
+(expect-failure
+  "guard rotation fails when ROTATE is not scoped"
+  "Managed capability not installed"
+  (rotate 'emily (read-keyset 'doug)))
+
+(env-sigs [{'key: "keys1", 'caps: [(coin.ROTATE (read-keyset 'doug))]}])
```

Comment:
So, the issue with sig caps is they can only be primitive pact values. I don't know how we resolve this with having `PactValue` in the `UserCapability` type, but it's something to deal with in Pact. Maybe we need to enforce this in `env-sigs`?

Reply:
We should probably parse and case on the pact value type